pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -225,6 +225,10 @@ public static class Broker {
     public static final String CONFIG_OF_BROKER_ID = "pinot.broker.instance.id";
     public static final String CONFIG_OF_BROKER_HOSTNAME = "pinot.broker.hostname";
     public static final String CONFIG_OF_SWAGGER_USE_HTTPS = "pinot.broker.swagger.use.https";
+    // Comma separated list of packages that contains javax service resources.
+    public static final String BROKER_RESOURCE_PACKAGES = "broker.restlet.api.resource.packages";
+    public static final String DEFAULT_BROKER_RESOURCE_PACKAGES = "org.apache.pinot.broker.api.resources";
```

Comment:
it's a bit weird to me that the controller resource package config is actually in `ControllerConf` class and i cant find an equivalent `BrokerConf` class.

Reply:
yeah I noticed that too. I started a draft locally to move everything to `BrokerConf`. But it's a much bigger change and will publish it separately.